src/macaron/repo_finder/repo_finder_java.py

```diff
@@ -24,6 +24,7 @@ class JavaRepoFinder(BaseRepoFinder):
     def __init__(self) -> None:
         """Initialise the Java repository finder instance."""
         self.pom_element: Element | None = None
+        self.repositories: list[str] | None = None
```

Comment:
I wonder where this attribute `repositories` is being used. I can't seem to find it :sweat_smile: